clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleHttpLoadBalancerAtomicOperation.groovy

```diff
@@ -118,6 +130,16 @@ class DeleteGoogleHttpLoadBalancerAtomicOperation  implements AtomicOperation<Vo
       def backendServiceName = GCEUtil.getLocalName(backendServiceUrl)
       task.updateStatus BASE_PHASE, "Retrieving backend service $backendServiceName..."
       BackendService backendService = compute.backendServices().get(project, backendServiceName).execute()
+
+      if (backendService?.backends) {
+        task.updateStatus BASE_PHASE, "Server groups still associated with Http(s) load balancer ${description.loadBalancerName}. Failing..."
+        throw new IllegalStateException("Server groups still associated with Http(s) load balancer: ${description.loadBalancerName}.")
```

Comment:
We currently don't change any of the server group metadata on load balancer deletion, so that assumption can get you into trouble if you clone the leftover server group -- there will be metadata entries on that server group for a load balancer that doesn't exist. If we disallow deleting a load balancer with server groups, you have to explicitly disable a server group to get into the same scenario.